src/OpenTelemetry.Exporter.Jaeger/Implimentation/Int128.cs

```diff
@@ -0,0 +1,53 @@
+// <copyright file="Int128.cs" company="OpenTelemetry Authors">
+// Copyright 2018, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+namespace OpenTelemetry.Exporter.Jaeger.Implimentation
+{
+    using System;
+    using System.Diagnostics;
+
+    public struct Int128
+    {
+        public static Int128 Empty = new Int128 { };
```

Comment:
Does this compile?
```suggestion
        public static Int128 Empty = default;
```

Reply:
thanks. I forgot about the `default` keyword and was racking my brain as I tried to find what to put here. D'oh!